cmake/sirf.__init__.py.in

```diff
@@ -1,4 +1,18 @@
 __version_major__ = '@VERSION_MAJOR@'
 __version_minor__ = '@VERSION_MINOR@'
 __version_patch__ = '@VERSION_PATCH@'
-__version__ = '@VERSION_MAJOR@.@VERSION_MINOR@.@VERSION_PATCH@'
\ No newline at end of file
+__version__ = '@VERSION_MAJOR@.@VERSION_MINOR@.@VERSION_PATCH@'
+
+from sirf.SIRF import *
+
+import os
+import importlib
+
+module_files = [f for f in os.listdir(os.path.dirname(__file__)) if f.endswith('.py') and f != '__init__.py']
+
+for module_file in module_files:
+    module_name = module_file[:-3]
+    try:
+        globals()[module_name] = importlib.import_module(f'.{module_name}', package=__name__)
+    except ImportError as e:
+        print(f"Error importing module {module_name}: {e}")
```

Comment:
Why do we do this?

It's up to the user to know which modules they want to use.

Reply:
yes I will change... thanks!